### ISSUE TITLE
SOLVED: SSL Issues

### ISSUE THREAD

**Issue by AaronKoller** [tycomo/generator-pwa-angular-core#3]
I am able to start with 'npm run build.prod' just fine.
However, when I try 'npm run build.dev' the follow error appears:

> Uncaught ReferenceError: polyfills_lib is not defined
>     at Object.<anonymous> (main.js:82)
>     at __webpack_require__ (main.js:20)
>     at Object.<anonymous> (main.js:3140)
>     at __webpack_require__ (main.js:20)
>     at Object.<anonymous> (main.js:3120)
>     at __webpack_require__ (main.js:20)
>     at Object.<anonymous> (main.js:3090)
>     at __webpack_require__ (main.js:20)
>     at Object.defineProperty.value (main.js:3009)
>     at __webpack_require__ (main.js:20)

**Comment by AaronKoller** [comment]
SOLVED: Never mind, I had disabled SSL in the startup.cs.  When I enabled it again, then 'dev' seems to work fine. 

If could be just me, but the command:
`set ASPNETCORE_ENVIRONMENT=Development`
seems to have no effect.  As Chrome shows an "SSL connection refused" message.

Using: 
`$Env:ASPNETCORE_ENVIRONMENT = "Development"`
Seems to do the trick as now the certificate is being loaded now.  After that I can use:
`dotnet run`

By the way, this is my favorite template on Yeoman!  Just seeing how you set things up has been a great learning experience!
